### PR TITLE
vmid parameter added

### LINKED ISSUE #5: [FEATURE] add vmid parameter
### Problem
I have two templates with the same name example on different nodes:
**id:** 140 Name: DebianBusterTemplate **node:** A
**id:** 130 Name: DebianBusterTemplate **node:** B

When i want to create a new clone on node B with clone_vm =  DebianBusterTemplate, this happen:

```
TASK [crear_kvm : Clone KVM in Proxmox node] *********************************************************************************************************************
fatal: [newClone -> B]: FAILED! => {"changed": false, "msg": "Unable to clone vm newClone from vmid 140=500 Internal Server Error: {\"data\":null}"}
```

### Possible solution
I have tested the vmid parameter with a value of 130 which is the template on my node and is working. A newClone with ID: 129 have been created.

The documentation says
`clone:Name of VM to be cloned. If vmid is setted, clone can take arbitrary value but required for initiating the clone.`

`vmid: Specifies the VM ID. Instead use name parameter.
If vmid is not set, the next available VM ID will be fetched from ProxmoxAPI.`

` newid: VMID for the clone. Used only with clone.
If newid is not set, the next available VM ID will be fetched from ProxmoxAPI.`
[source](https://docs.ansible.com/ansible/latest/modules/proxmox_kvm_module.html#parameters)

The documentation is not clear but i think the vmid is the id of the VM source when cloning.

Edit: this example on the documentation should help us understand the function of vmid.

```
# Clone VM with source vmid and target newid and raw format
- proxmox_kvm:
    api_user    : root@pam
    api_password: secret
    api_host    : helldorado
    clone       : arbitrary_name
    vmid        : 108
    newid       : 152
    name        : zavala  # The target VM name
    node        : sabrewulf
    storage     : LVM_STO
    format      : raw
    timeout     : 300  # Note: The task can take a while. Adapt
```

### PR DESCRIPTION
**Pull request by Dkmarce**
Closes #5

### CODE REVIEW

File: defaults/main.yml [hunk 1]
```diff
@@ -15,6 +15,7 @@ clone_vm: DebianBusterTemplate    # The VM to clone
 clone_storage: local-lvm          # Target storage for full clone.
 clone_format: raw                 # Target drive's backing file's data format. Used only with clone
 clone_target: "{{ node }}"        # Target node 
+clone_vmid: 1                     # VM source id
```
Comment (santiagomr) [inline defaults/main.yml]: All right. Maybe it's a good time to also assign a var to the `newid` argument
```yaml
clone_newid: 9
```

File: tasks/clone_kvm.yml [hunk 2]
```diff
@@ -14,5 +14,6 @@
     format: "{{ clone_format | default (omit) }}"       # Target drive's backing file's data format. Used only with clone
     timeout: 500                                        # Cloning can take a while
     target: "{{ clone_target }}"                        # Target node 
+    vmid: "{{ clone_vmid | default (omit) }}"           # VM source id
```
Comment (santiagomr) [inline tasks/clone_kvm.yml]: And invoke that var over here:

```yaml
newid: "{{ clone_newid | default (omit) }}"
```

File: defaults/main.yml (hunk 1 above)
Comment (andrespias) [reply defaults/main.yml]: I agree, regardless of the documentation, are you sure if this parameter works? does it work to indicate the new LXC id to the cluster?

Comment (Dkmarce) [reply defaults/main.yml]: I've tested the `newid` parameter and it does work.
I tested the `vmid` parameter this morning too.

File: tasks/clone_kvm.yml (hunk 2 above)
Comment (Dkmarce) [reply tasks/clone_kvm.yml]: Thank you.

**Pull request merged**